### ISSUE TITLE
Add support Python 3.9

### ISSUE THREAD

**Issue by CrazyLionHeart** [0101/pipetools#15]
According alert from DeprecationWarning we have two issues:

```
pipetools/main.py:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import Iterable
```

and

```
pipetools/utils.py:2: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import Mapping
```

Docs: https://docs.pytest.org/en/stable/warnings.html

**Comment by 0101** [comment]
Thanks for the report. That should be easy to fix. Do you want to make a pull request? :)

**Comment by 0101** [comment]
@CrazyLionHeart Fix is published in version 0.3.6
